.travis.yml

```diff
@@ -1,25 +1,37 @@
-language: c
+language: cpp
```

Comment:
I found hard with Travis to make it pass on all configurations and I'm afraid I didn't find a better solution.